### ISSUE TITLE
something is missing

### ISSUE THREAD

**Issue by qianxiaqiyue** [YongshengPan/ASEG-for-PET2CT#2]
Another problem GANEverageThing.layers is missing, build_resnet_block and build_resnet_block3d as well as cv2 are missing!

**Comment by zsq964542358** [comment]
yes, I also find there is something missing, I am very confused, wish author can solve it.
